docs/essentials/viewport.md

```diff
@@ -128,3 +128,11 @@ You can change your story through [parameters](../writing-stories/parameters.md)
 />
 
 <!-- prettier-ignore-end -->
+
+### Keyboard shortcuts
+
+* Previous viewport: `shift + v`
```

Comment:
Should this be using the `kbd` element instead?

For example: <kbd>ctrl</kbd> + <kbd>V</kbd>